flight_phase_files/HALO_RF04_20200126_info.yaml

```
@@ -21,71 +26,366 @@ remarks:
 - Four coordinated straight legs over Meteor.
 - Cirrus clouds in 14-17 km present for almost full flight.
 segments:
-- kind: straight_leg
+- kinds:
+  - straight_leg
   name: super_curtain
+  irregularities: []
+  segment_id: HALO-0126sc1
   start: 2020-01-26 12:07:06
-  end: 2020-01-26 12:15:10
-- kind: circle
-  name: c1
-  start: 2020-01-26 12:27:11
-  end: 2020-01-26 13:17:15
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-26 13:17:15
-  end: 2020-01-26 13:38:57
-- kind: circle
-  name: c2
-  start: 2020-01-26 13:38:57
-  end: 2020-01-26 14:27:36
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-26 14:27:36
-  end: 2020-01-26 14:43:29
-- kind: circle
-  name: c3
-  start: 2020-01-26 14:43:29
-  end: 2020-01-26 15:35:58
-- kind: straight_leg
-  name: leg1_over_Meteor
+  end: 2020-01-26 12:17:10
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: Circle-1
+  irregularities: []
+  segment_id: HALO-0126c1
+  start: 2020-01-26 12:26:08
+  end: 2020-01-26 13:22:28
```

Start and end overlap a bit; end circle two seconds earlier
```suggestion
  end: 2020-01-26 13:22:26
```